components/scream/cime_config/buildnml

```diff
@@ -12,7 +12,7 @@ sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))
 from standard_script_setup import *
 from CIME.case import Case
 from CIME.utils import expect, run_cmd_no_fail, safe_copy
-from CIME.buildnml import create_namelist_infile, parse_input
+from CIME.buildnml import create_namelist_infile, parse_input, build_xcpl_nml
```

Comment:
Where is this new import used?

Reply:
I'm not sure.  It looks like xcpl_nml might refer to the "dead" component stuff but I can't remember putting this in.  I may have been copying from an example.  I'll remove this and see if it still works.